### PR TITLE
add function on input genre etc

### PR DESCRIPTION
**Pull request by miyabitanimchi**
I added a function to show all the rock related genres when we search for rock on input.
Also, I found one bug related the filter function like the example below.

e.g., User A searches for a genre rock **by inputting** => the result shows : user B, C, D => user A browser back => user A searches for indie rock with the button => the result still shows the previous ones, user B, C, D.

I fixed it by adding removeItem() for the session storage on `filter2/MusicGenre.js`

### CODE REVIEW

**Review by Kit486759 — APPROVED** [review]
It works great. Thank you @miyabitanimchi  for the fix.

**Review by BlackStone8960 — APPROVED** [review]
It looks good, thank you!

File: src/components/filter/FilterByInputGenre.js [hunk 1]
```diff
@@ -5,18 +5,46 @@ import { Button } from "@material-ui/core";
 
 const FilterByInputGenre = () => {
   const [inputGenre, setInputGenre] = useState("");
+  const [firstTimeFlag, setFirstTimeFlag] = useState(true);
+  console.log(firstTimeFlag);
```
Comment (BlackStone8960) [inline src/components/filter/FilterByInputGenre.js:9]: Can you just delete this line to execute console.log()?
Thank you!

**Review by YumiKimbara — APPROVED** [review]
It looks good, thank you!!

**Pull request merged**